kolibri_instant_schools_plugin/models.py

```diff
@@ -11,7 +11,7 @@
 
 class AboutFAQ(models.Model):
     html = models.TextField()
-    kind = models.TextField(choices=(("FAQ", "FAQ"), ("About", "About")))
+    kind = models.TextField(choices=(("FAQ", "FAQ"), ("About", "About"), ("Terms", "Terms")))
```

Comment:
AboutFAQ might not have been the best name for this in retrospect now that I've added a third choice 🤷🏻‍♂️

Reply:
Could be worse, we might be FAQing About!